frontend/apps/web/src/FormsApp.tsx

```diff
@@ -50,6 +50,9 @@ export const FormsApp = (props: {}) => {
 			fromAPIRawValue: _ => _ == undefined ? List() : List(_),
 			toAPIRawValue: _ => _.valueSeq().toArray()
 		},
+		"base64File": { fromAPIRawValue: _ => typeof _ == "string" ? _ : "", toAPIRawValue: _ => _ },
+		"secret": { fromAPIRawValue: _ => typeof _ == "string" ? _ : "", toAPIRawValue: _ => _ },
```

Comment:
Though the API should never return the secret, i think we should also encode that here, so fromAPIRawValue should always parse to an empty string.